libbeat/processors/add_cloud_metadata/_meta/fields.yml

```diff
@@ -53,4 +52,12 @@
       path: cloud.region
       migration: true
 
+    - name: azure.resourcegroup
```

Comment:
Is this consistent with what we have been using in Fleet integrations? I think I recall `azure.resource.group` being used over there.

Reply:
I am also a little concerned about this processor beginning to produce fields outside of the ECS `cloud` and `orchestrator` namespaces.

Downstream consumers (e.g Fleet integrations) should ideally be updated to have this field included in their mappings since it's not part of ECS. This affects almost all integrations since Agent includes `add_cloud_metadata` with most input types from Beats. (Imagine if we ran our CI on Azure for Fleet integration system testing, after this merges every integration's system tests would begin failing because this field was not defined in its mappings.)

The other concern is whether or not existing pipelines will safely handle the addition of this field. For example if an ingest pipeline is already using this field then does it safely handle the case where the field already exists.

Reply:
The `azure.resource.group` , created by a pipeline? as I noticed is the resource group stripped. It only includes the group's name. The addition of the `azure.resourcegroup.name` field I did was to me a nice to have and definitely not the reason of this PR.
I understand that it then needs to be added in all the integrations because it comes from the processor.
Also it is confusing having two similar named fields. I chose the naming to be compliant with Otel.

I hear your concerns and I will remove this field as it beats the purpose of this PR which was to add the orchestrator ecs fields.

Reply:
> The azure.resource.group , created by a pipeline?

I'm not familiar with these integrations, but, yes, I see it being created by a grok in a few pipeline.

https://github.com/search?q=repo%3Aelastic%2Fintegrations+path%3A**%2F*.yml+%2Fazure%5C.resource%5C.%3Fgroup%2F&type=code

> I understand that it then needs to be added in all the integrations because it comes from the processor.

This is a really general problem we have with Fleet integrations in that there is no easy or centralized way to account for the fields being added by the processors.

> I hear your concerns and I will remove this field as it beats the purpose of this PR which was to add the orchestrator ecs fields.

👍